### DISCUSSION TITLE
Welcome to GataBot-MD Discussions!

### DISCUSSION THREAD

**Discussion by GataNina-Li** [GataNina-Li/GataBot-MD#84] [Announcements]

**Comment by moreno123zjzn** [comment]
Holas

**Comment by moreno123zjzn** [comment]
Uhhhh intente instalar el chat bots pero no puede a casi lo último dice error

**Reply by DabiCZNG** [reply]
Tienes que tener paciencia ya que asi sale al final siempre pero si conecta te lo digo porque pasa con casi todos los repositorio de bots para WhatsApp
